### ISSUE TITLE
API: Add functionalities

### ISSUE THREAD

**Issue by tgouverneur** [tgouverneur/SPXOps#30]
Waiting for input.

**Comment by gmsoft-tuxicoman** [comment]
For lsVM, can you add a filter to match VMs which aren't defined anymore for a few days ?

**Comment by tgouverneur** [comment]
Yes, though it would only be for a limited amount of time (basically until they get also deleted from spxops' db), would that fit?

**Comment by gmsoft-tuxicoman** [comment]
Yes, I would be doing this query every hour. So if it stays there for more than that, it should be good.
